test/benchmark/fuse_db/bin/sqlite_verify.py

```
@@ -0,0 +1,68 @@
+if ic != "ok": ok = False
+
+cnt    = con.execute("SELECT COUNT(*) FROM t;").fetchone()[0]
+minid  = con.execute("SELECT MIN(id) FROM t;").fetchone()[0]
+maxid  = con.execute("SELECT MAX(id) FROM t;").fetchone()[0]
+sum_chk= con.execute("SELECT COALESCE(SUM(chk),0) FROM t;").fetchone()[0]
+sum_id = con.execute("SELECT COALESCE(SUM(id),0)  FROM t;").fetchone()[0]
+
+if mode == "exact":
+    target = expected_rows
+    if cnt != expected_rows:
+        print(f"rows: got={cnt} expected={expected_rows}  MISMATCH"); ok = False
```

Ruff (0.15.15)

[error] Multiple statements on one line (semicolon)

(E702)